steps/export.ts

```diff
@@ -169,11 +171,11 @@ const exportTarget = async (target: Target, bananaI18n: Banana) => {
     {
       fileName: `./dist/${target.output}.zim`,
       welcome: 'index.html',
-      fullTextIndexLanguage: languageCode,
+      fullTextIndexLanguage: iso6393LanguageCode,
```

Comment:
Shouldn't this be `getISO6393(target.languages[0])` instead? I think `iso6393LanguageCode` would contain `mul` which would disturb indexing.